src/main.c

```diff
@@ -1055,6 +1069,21 @@ static int open_inet_listener(h2o_configurator_command_t *cmd, yoml_t *node, con
         if (setsockopt(fd, SOL_SOCKET, SO_REUSEADDR, &flag, sizeof(flag)) != 0)
             goto Error;
     }
+
+   {
+        int flag = 1;
+
+        if (timestamping) {
+#if defined(__linux__)
+            if (setsockopt(fd, SOL_SOCKET, SO_TIMESTAMPNS, &flag, sizeof(flag)) != 0) {
```

Comment:
Would this feature available on virtually every linux box (e.g., old kernels, virtualized environment)?

If that is not the case (or if we are uncertain), it might be better to just print the same error message as below and continue.